error-handlers/error-handlers.js

```diff
@@ -0,0 +1,16 @@
+const handle400 = (err, req, res, next) => {
+  console.log(err);
```

Comment:
Remember to remove console.logs once you're done debugging. (Though the 500 error handler could be the one exception to this rule!)